core/src/main/java/io/grpc/internal/AbstractServerStream.java

```diff
@@ -170,6 +170,11 @@ public final boolean isReady() {
   }
 
   @Override
+  public void setListener(ServerStreamListener serverStreamListener) {
```

Comment:
Go ahead and mark this as `final`, to help make it clear that we know of no reason that an implementation would need to override this method. (If there is a need later, we can always remove the `final`).